Revit_Core_Engine/Compute/Warnings.cs

```diff
@@ -324,7 +324,7 @@ internal static void ElementTypeNotFoundWarning(this IBHoMObject iBHoMObject)
             string message = "Element type has not been found for given BHoM Object. Please make sure the name of the BHoM object (or its type property, e.g. Construction) is matching the Revit family type name.";
 
             if (iBHoMObject != null)
-                message = string.Format("{0} BHoM Guid: {1}", message, iBHoMObject.BHoM_Guid);
+                message = $"{message}\nObject name: {iBHoMObject.Name}, object type: {iBHoMObject.GetType().FullName}, BHoM_Guid: {iBHoMObject.BHoM_Guid}";
```

Comment:
```suggestion
                message = $"{message}\nObject name: {iBHoMObject.Name}, object type: {iBHoMObject.GetType().FullName}";
```

if you cut out the GUID, grasshopper will collapse several identical messages into one- i.e. one message for all the bars with each unique sectionProperty that doesn't have a corresponding familyType in the Revit model. A GUID is only helpful if you can pipe it to some logic which will select those objects, and since error messages are for humans, I don't see a reason to include it

Reply:
Agreed with your comment @JosefTaylor - the Guid does not sound very helpful in case the name and type are provided. I have deleted the requested section.

Actually, this is a more universal problem - the warnings and errors in Revit_Toolkit are far from perfect, often too vague or hardly helpful for the non-expert user. #400 has been raised 2 years ago, still waiting for the right moment 😢